tests/python/test_sparse_matrix.py

```diff
@@ -9,7 +9,7 @@
                                                    (ti.f32, 'row_major'),
                                                    (ti.f64, 'col_major'),
                                                    (ti.f64, 'row_major')])
-@test_utils.test(arch=ti.cpu)
+@test_utils.test(arch=ti.cpu, offline_cache=False)
```

Comment:
Hi @PGZXB , I get errors when I set `offline_cache = True`, do you have any clue?

Reply:
Hi @FantasyVR. Could you describe the error in more detail?
It would be great if you could provide the output of the failed testing, which will be very helpful to me.

Reply:
Hi @PGZXB, when I set `offline_cache = True` in `misc/sparse_matrix.py`, the output is different from when `offline_cache= False` which output correct results.

Reply:
@FantasyVR I removed all `offline_cache=False` in your PR and built the branch.
Then:
* `python tests/run_tests.py -a cpu`: PASSED
* `python tests/run_tests.py -a cpu --with-offline-cache`: PASSED

I think that you could remove all `offline_cache=False` in your PR and commit. 
Then see the CI's report. 
I will help you if any errors about the offline cache occur.

Reply:
Thanks @PGZXB, I will see if this PR can pass CI after removing `offline_cache=False`.